cluster/gce/util.sh

```diff
@@ -824,6 +824,10 @@ function construct-linux-kubelet-flags {
     flags+=" --container-runtime-endpoint=${CONTAINER_RUNTIME_ENDPOINT}"
   fi
 
+  if [[ ${ENABLE_CREDENTIAL_SIDECAR:-false} == "true" ]]; then
+    flags+=" --image-credential-provider-config=/etc/srv/kubernetes/cri_auth_config.yaml --image-credential-provider-bin-dir=/home/kubernetes/bin"
```

Comment:
Ignoring the where we log issue for now, is the level of logging currently sufficient to debug the issue/identify the request? My guess is we need to set the log level to 2 or 3 to do that.

Reply:
I had it at the default level and that seemed to work fairly well for getting an empty cluster up and running, but I imagine for deeper issues an increased log level would certainly help - I've added a `--v=3` argument to the plugin.